internal/schema/sample_bucket_test.go

```diff
@@ -0,0 +1,70 @@
+package schema
+
+import (
+	"testing"
+
+	"github.com/couchbasecloud/terraform-provider-couchbase-capella/internal/errors"
+
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/stretchr/testify/assert"
```

Comment:
incorrect formatting

Reply:
What do we use to check the formatting of the imports? Shouldn't the linter be picking up these types of issues?